src/pages/docs/step-ca/policies.mdx

```diff
@@ -0,0 +1,295 @@
+A policy consists of one or more rules that are all evaluated just before a new certificate is signed.
+Upon evaluation of the rules all names in a certificate request are checked to be allowed.
+If one of the names is not allowed, the certificate will not be signed.
+An error will be reported to the client indicating why the certificate wasn't created.
+
+You can configure a policy for X.509 certificates, SSH user certificates and SSH host certificates.
+The CA will evaluate only the policy corresponding to the type of certificate that is requested.
+This means that rules for X.509 certificates will not be evaluated when an SSH user or host certificate is requested.
+
+<Alert severity="info">
+  <AlertTitle>Need more control?</AlertTitle>
+  <div>
+    A self-hosted <Code>step-ca</Code> instance can only be configured with a policy on the authority level.
+    In a <Link
+      external
+      href="https://smallstep.com/signup?product=cm"
+    >
+      free hosted smallstep Certificate Manager authority
+    </Link>, policies can be configured on the authority, on each provisioner, and on every ACME account, providing more granular control.
+  </div>
+</Alert>
+
+## Policy Evaluation
+
+Policies consist of one or more rules, each resulting in a (range of) names to be allowed or denied.
+A rule is applied to a single name type, like X.509 DNS domains or IP addresses.
+You'll need to configure multiple rules if you want the policy to apply to multiple types of names.
+
+If a rule is configured to allow a certain type of name, all other types of names are automatically denied.
+If a rule is configured to deny a certain type of name, all other types of names are still allowed.
+Generally a policy will contain a set of allowed names and can contain a number of exclusions.
+
+A rule can be configured to allow a range of names using a wildcard notation corresponding to the type of name.
+Another rule can be configured that denies a subset of the allowed range of names.
+Creating the opposite set of rules, denying a certain range and allowing a subset of that range, however, will not result in the names to be allowed. 
+
+The semantics used for evaluating rules and matching requested certificate names is described in the following sections.
+
+
+## X.509 Policies
+
+A policy for X.509 certificates can have rules for the following types of names:
+
+- Domain Names
+- IP Addresses
+- Email addresses
+- URIs
+- Subject Common Names
+
+The following subsections describe the specifics for each of those.
+
+### Domain Names
+
+Rules for DNS domains are evaluated to match largely according to how [Name Constraints (RFC 5280)](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.10)
+are evaluated, but applied to leaf certificates instead of CA certificates:
+
+- Domain names are normalized into their [internationalized representation (RFC 5890)](https://datatracker.ietf.org/doc/html/rfc5890).
```

Comment:
would it make sense to give an example of an internationalized domain name here?

Reply:
Added a short paragraph about this in 2a68d5e6426ee6be71765e750ccd805d075ae293.